content/rancher/v2.x/en/installation/ha/rancheronk8s/_index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Rancher HA on Kubernetes (Comming soon)
+weight: 2300
+draft: true
+version: v2.0
+lang: en
+---
+
+---
+
+**Requirements**
+
+- running k8s cluster.
+- rancher cli or helm command in OS path.
+- kubectl cli command in OS path.
+- TLS mode `external`: 
+  - External LB running with SSL.
+  - FQDN dns entry for rancher server pointing to external LB.
+- TLS mode `ingress`: 
+  - FQDN dns entry for rancher server pointing to k8s cluster nodes running ingress controller.
+  - TLS issuer `<blank>`: TLS key and certificate. ca cert if selfsigned cafiles for your FQDN.
+  - TLS issuer `ca`: CA key and cert.
+  - TLS issuer `acme`: Letsencrypt email ID.
+    
+
+**K8S cluster**
+
+Rancher v2.0 could be deployed in a k8s cluster. It's recommended a minimal of 3 nodes k8s cluster with overlapped planes.
+
```

Comment:
For a different configuration, ie. non-overlapping, point out that etcd and K8s masters need to run with at least the following to make them resilient:
- etcd running on at least 3 hosts
- K8s master running on at least 2 hosts

Reply:
Just talking about minimal requirement. 
Added recommended 7 nodes k8s cluster with separated planes.
- 3 nodes for etcd plane.
- 2 nodes for control plane.
- 2 nodes for worker plane.